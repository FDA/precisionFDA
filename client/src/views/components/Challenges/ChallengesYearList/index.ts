import { YearList } from '../../List/YearList'
import { queryChallengesYearList } from '../../../../api/challenges'


class ChallengesYearList extends YearList {
  static defaultProps = {
    elementName: 'challenges',
    query: queryChallengesYearList,
    setYearHandler: () => {},
  }
}

export {
  ChallengesYearList,
}
export default ChallengesYearList
