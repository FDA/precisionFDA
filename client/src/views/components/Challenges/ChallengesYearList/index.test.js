import React from 'react'
import { shallow } from 'enzyme'

import ChallengesYearList from './index'


describe('ChallengesYearList', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<ChallengesYearList setYearHandler={() => {}} />)

    expect(wrapper).toMatchSnapshot()
  })
})