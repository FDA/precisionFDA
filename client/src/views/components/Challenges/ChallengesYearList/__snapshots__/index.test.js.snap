// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChallengesYearList matches snapshot 1`] = `
<QueryList
  className="year-list"
  emptyMessage="No previous challenges"
  listExtractor={[Function]}
  query={[Function]}
  template={[Function]}
/>
`;
